src/de/mod.rs

```diff
@@ -8,6 +8,9 @@ use serde_crate::de::IntoDeserializer;
 use serde_crate::de::Error as DeError;
 use ::SizeLimit;
 use super::{Result, Error, ErrorKind};
+use self::read::BincodeRead;
+
+pub(crate) mod read;
```

Comment:
Ah yep, this was mainly added just to just so I wouldn't have to think about it when I was prototyping the new module structure.  I'll add this to my list of TODOs at the PR description.